index.js

```diff
@@ -71,8 +71,10 @@ module.exports = {
           req.body._ids = self.apos.launder.ids(req.body._ids);
 
           const extension = self.apos.launder.string(req.body.extension);
-          const batchSize = self.apos.launder.integer(req.body.batchSize);
-          const expiration = typeof self.options.export === 'object' ? self.apos.launder.integer(self.options.export.expiration) : null;
+          const batchSize = typeof self.options.export === 'object' &&
```

Comment:
laundering module config options is not necessary. Both of these need reasonable defaults.